cgroup_top.py

```diff
@@ -631,6 +679,8 @@ def on_keyboard(c):
             run(-2, ['docker', 'stop', '-t', '0', selected_name])
         elif selected['type'] in ['lxc', 'lxc-user'] and HAS_LXC:
             run(selected['owner'], ['lxc-stop', '-k', '--name', selected_name, '--nowait'])
+        elif selected['type'] == 'openvz' and HAS_OPENVZ:
+            run(selected['owner'], ['vzctl', 'stop', selected_name])
```

Comment:
add --fast flag - it not halt from inside container, it kill it at once without wait-timeout.